SPECS/sos/sos.spec

```diff
@@ -46,13 +48,17 @@ install -d -m 700 %{buildroot}%{_sysconfdir}/%{name}/cleaner
 install -d -m 755 %{buildroot}%{_sysconfdir}/%{name}/presets.d
 install -d -m 755 %{buildroot}%{_sysconfdir}/%{name}/groups.d
 install -d -m 755 %{buildroot}%{_sysconfdir}/%{name}/extras.d
+install -d -m 755 %{buildroot}%{_tmpfilesdir}
 install -m 644 %{name}.conf %{buildroot}%{_sysconfdir}/%{name}/%{name}.conf
+install -m 644 tmpfiles/tmpfilesd-sos-rh.conf %{buildroot}%{_tmpfilesdir}/%{name}.conf
```

Comment:
I think Eric suggested this tmpfiles config is not something we want? I am certainly baffled as to why RH wants this config in their distro...

Reply:
Yeah, synced with Eric and will remove this